scale/docs/architecture/triggers.rst

```diff
@@ -38,30 +43,39 @@ every file ingest and checked against all ingest trigger rules.
    }
 
 The *condition* field is used to define the conditions for when the ingest rule is triggered. The *media_type* field
-says that an ingested file must have a media type of *text/plain* (a plain text file) in order to trigger this rule. The
-*data_types* field specifies that the ingested file must also have the data types "foo" and "bar" tagged on it in order
-to trigger the rule. The *data* field specifies the information needed to create the applicable job/recipe (whatever the
+says that an ingested file must have a media type of *text/plain* (a plain text file) in order to trigger this rule. The 
+three data type filtering fields (*data_types*, *any_of_data_types*, *not_data_types*) stipulate that the parsed file must 
+have the data type "foo" tagged, may have data type "bar" tagged, and can not have data type "baz" tagged in order to 
```

Comment:
The language 'may have data type "bar" tagged' is incorrect, in this case it must have "bar" (as it's the only tag listed). I would recommend having two tags in this field and have language indicating that the file must match at least one of them.